src/font.ts

```diff
@@ -199,15 +187,15 @@ export class Font {
     let size: string;
     const sz = fontInfo.size;
     if (sz === undefined) {
-      size = Font.SIZE + 'pt ';
+      size = '';
```

Comment:
Same here. We want to have this method always return a valid CSS font declaration. The default size should be something like '10pt' or '12pt'. Hardcoding it is fine if you don't like the constants.

Reply:
done